python/lsst/meas/algorithms/psfCandidate.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+
+from ._psfCandidate import PsfCandidateF
+
+__all__ = []  # import only for side effects
+
+PsfCandidateF.getCandidateRating = PsfCandidateF._getCandidateRating
```

Comment:
I still haven't the foggiest idea why this is being done. Why not just get rid of the Python and define "getCandidateRating" here? All the python does is call the C++ version, isn't it? What am I missing.